CHANGELOG.next.asciidoc

```diff
@@ -71,6 +71,26 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Metricbeat*
 
 - Add _bucket to histogram metrics in Prometheus Collector {pull}11578[11578]
+- Fix panics in vsphere module when certain values where not returned by the API. {pull}9784[9784]
```

Comment:
Rebase issue?

Reply:
yes, looks like it , fixed it in the new commit , also addressed the review comments bellow